bioimageio/core/build_spec/build_model.py

```diff
@@ -258,6 +266,160 @@ def _get_dependencies(dependencies, root):
     return model_spec.raw_nodes.Dependencies(manager=manager, file=_process_uri(path, root))
 
 
+def _get_deepimagej_macro(name, kwargs, export_folder):
+
+    # macros available in deepimagej
+    assert name in ("scale_linear", "scale_range", "zero_mean_unit_variance")
+    # TODO deep imagej has a binarize macro but I have no idea what it is doing
+    # assert name in ("binarize", "scale_linear", "scale_range", "zero_mean_unit_variance")
+
+    if name == "scale_linear":
+        macro = "scale_linear.ijm"
+        replace = {"gain": kwargs["gain"], "offset": kwargs["offset"]}
+
+    elif name == "scale_range":
+        macro = "per_sample_scale_range.ijm"
+        replace = {"min_precentile": kwargs["min_percentile"], "max_percentile": kwargs["max_percentile"]}
+
+    elif name == "zero_mean_unit_variance":
+        mode = kwargs["mode"]
+        if mode == "fixed":
+            macro = "fixed_zero_mean_unit_variance.ijm"
+            replace = {"paramMean": kwargs["mean"], "paramStd": kwargs["std"]}
+        else:
+            macro = "zero_mean_unit_variance.ijm"
+            replace = {}
+
```

Comment:
```suggestion
    else:
        raise ValueError(name)

```
would save you the duplication of available macro names in the assert above

Reply:
But the error message is more meaningful in the other case. I will update a bit differently.